tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -269,7 +269,11 @@ static uint8_t supported_commands(const void *cmd, uint16_t cmd_len,
 		tester_set_bit(rp->data, BTP_GAP_SET_EXTENDED_ADVERTISING);
 	}
 
-	*rsp_len = sizeof(*rp) + 4;
+	if (IS_ENABLED(CONFIG_BT_RPA_TIMEOUT_DYNAMIC)) {
```

Comment:
would be good to have comment about which octet this is (as above), this make it easier to figure out rsp_len